### PR TITLE
tests and fixture added

### PR DESCRIPTION
**Pull request by DmytroSvirsa**

### CODE REVIEW

File: taxi_service_db_data.json [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+      "is_superuser": false,
+      "username": "bob.newby",
+      "first_name": "Bob",
+      "last_name": "Newby",
+      "email": "bob.newby@taxi.com",
+      "is_staff": false,
+      "is_active": true,
+      "date_joined": "2022-08-09T14:19:09Z",
+      "license_number": "BOB09631",
+      "groups": [],
```
[the remark this answers is not in the document]
Comment (DmytroSvirsa) [reply taxi_service_db_data.json:319]: +

File: tests/test_taxi-service-user-form-and-validation.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.forms import DriverLicenseUpdateForm
+
+
+class ValidLicenseNumberFormTests(TestCase):
+    def test_validation_license_number_with_valid_data(self):
+        test_license_number = "TES12345"
+        form = DriverLicenseUpdateForm(
```
[the remark this answers is not in the document]
Comment (DmytroSvirsa) [reply tests/test_taxi-service-user-form-and-validation.py:11]: agry

File: README.md [hunk 3]
```diff
@@ -2,15 +2,24 @@
 
 - Read [the guideline](https://github.com/mate-academy/py-task-guideline/blob/main/README.md) before start.
 - Make sure that you change the settings for [html-files](https://github.com/mate-academy/py-task-guideline/blob/main/html_settings/README.MD).
+- Use the following command to load prepared data from fixture to test and debug your code:
+  
+`python manage.py loaddata taxi_service_db_data.json`
+
+- After loading data from fixture you can use following superuser (or create another one by yourself):
```
[the remark this answers is not in the document]
Comment (DmytroSvirsa) [reply README.md:9]: it not he first task like this, they learn how to do it before, not shure about that

File: taxi_service_db_data.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (DmytroSvirsa) [reply taxi_service_db_data.json:319]: yep

File: taxi_service_db_data.json [hunk 4]
```diff
@@ -0,0 +1,503 @@
...
+      "name": "BAIC",
+      "country": "China"
+    }
+  },
+  {
+    "model": "taxi.manufacturer",
+    "pk": 14,
+    "fields": {
+      "name": "BMW",
+      "country": "Germany"
```
[the remark this answers is not in the document]
Comment (DmytroSvirsa) [reply taxi_service_db_data.json:111]: uf, it must be changed not only here but in all tasks, may be in future

**Review by Nattalli — APPROVED** [review]
Now it looks better 👍🏻

**Comment by danylott** [comment]
We need to think, how would we check this part: 
image
It's quite important, and students will make errors with it.
Ofc. we may leave it as it is (without tests), but maybe you have some ideas?

**Comment by Nattalli** [comment]
> We need to think, how would we check this part: image It's quite important, and students will make errors with it. Ofc. we may leave it as it is (without tests), but maybe you have some ideas?

We can clarify the URL for this button and then just make some simple tests and check that the db was changed(as it should be saved in some field)

**Pull request merged**